.agent/skills/reflect/scripts/gather_evidence.sh

```
@@ -0,0 +1,27 @@
+echo ""
+echo "## Uncommitted"
+git status --short 2>/dev/null || echo "(not in a git repo)"
+
+echo ""
+echo "## Environment Hints"
+echo "### .env files (existence only, no values)"
+find . -maxdepth 2 -name '.env*' -not -path '*/node_modules/*' 2>/dev/null | head -10
+
+echo ""
+echo "### Dev server config"
+grep -l 'dev.*server\|"dev"' package.json */package.json 2>/dev/null | head -5
```

Shellcheck (0.11.0)

[info] Use ./*glob* or -- *glob* so names with dashes won't become options.

(SC2035)